### PR TITLE
Ekf optimalization

### PR DESCRIPTION
**Pull request by rafal-gorecki**
### EKF
After the corrections, ekf ​​should generate such data, but for completeness you can do it in a circle, because I prepared the data from rosbag.

Additionally, the bag is quite simple, so it is difficult to compare individual configurations.

image

### Simulation 
**topic remap** - I also corrected the discrepancy in topics between the simulation and the robot.
**wrong units** - There was wrong values in simulated IMU `g/s^2` but should be `m/s^2`

### CODE REVIEW

**Comment by delihus** [comment]
There will be conflicts due to https://github.com/husarion/panther_ros/pull/236. In the https://github.com/husarion/panther_ros/pull/236, the IMU covariances are implemented from the documentation of the sensor and the Madgwick filter.

File: panther_controller/config/WH04_controller.yaml [hunk 1]
```diff
@@ -36,7 +41,7 @@
       publish_rate: 100.0
       odom_frame_id: odom
       base_frame_id: base_link
-      twist_covariance_diagonal: [1.3e-6, 2.7e-7, 0.0, 0.0, 0.0, 9.3e-6] # Selected intuitively based on WH01 results
+      twist_covariance_diagonal: [1.1e-6, 1.1e-6, 0.0, 0.0, 0.0, 3.8e-4] # Selected intuitively based on WH01 results
```
Comment (rafal-gorecki) [inline panther_controller/config/WH04_controller.yaml]:
```suggestion
      twist_covariance_diagonal: [1.1e-6, 1.1e-6, 0.0, 0.0, 0.0, 1.0e-5] # Selected intuitively based on WH01 results
```

File: panther_controller/config/WH04_controller.yaml [hunk 2]
```diff
@@ -36,7 +41,7 @@
       publish_rate: 100.0
       odom_frame_id: odom
       base_frame_id: base_link
-      twist_covariance_diagonal: [1.3e-6, 2.7e-7, 0.0, 0.0, 0.0, 9.3e-6] # Selected intuitively based on WH01 results
+      twist_covariance_diagonal: [1.1e-6, 1.1e-6, 0.0, 0.0, 0.0, 1.0e-5] # Selected intuitively based on WH01 results
```
Comment (rafal-gorecki) [inline panther_controller/config/WH04_controller.yaml]:
```suggestion
      twist_covariance_diagonal: [2.7e-5, 2.7e-5, 0.0, 0.0, 0.0, 1.0e-4] # Selected intuitively based on WH01 results
```

File: panther_bringup/config/ekf.yaml [hunk 3]
```diff
@@ -1,13 +1,14 @@
 /**:
   ekf_node:
+    # Default
```
Comment (rafal-gorecki) [inline panther_bringup/config/ekf.yaml]:
```suggestion
```

**Review by KmakD — APPROVED** [review]

**Pull request merged**